lib/puppet/application/ssl.rb

```diff
@@ -140,6 +140,10 @@ def main
     when 'verify'
       verify(certname)
     when 'clean'
+      possible_extra_args = command_line.args.drop(1)
+      unless possible_extra_args.empty?
+        raise Puppet::Error, _("Extra arguments detected: %{args}") % { args: possible_extra_args.join(' ') }
```

Comment:
Should we add some text like did you mean to run `puppetserver ca clean --certname <name>` or `puppet ssl clean --target <name>`?